### DISCUSSION TITLE
Do not allow land painting while clicking on map tool

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#20219] [Feature requests] [answered]
Please change it so when you open the map with the land tool open, clicking in the map window does not paint the terrain. I accidentally painted the whole terrain on multiplayer when I was trying to view the map and drag it around with the mouse cursor.

**Comment by TheClaudeQC** [comment]
I don't think it's a good idea because then it will be harder for scenario builders to make big section of desert, swamps, forests, etc. on their massive scenario of 200+ tiles in each directions.

Also that since a long time, openrct2 have a bug in 2 borders of the map, the 2 uppers one (default rotation) have terrain issue, as it doesn't allow both park bounderies nor terrain soils to be changed so close to the border...

**Reply by 733737** [reply]
I didn't think of that. Maybe it could be an option in the options menu that also allows for it to be configured to only draw in the map window after an initial click in the map window, or to only draw the traditional way in single player, etc.

**Reply by Gymnasiast** [reply]
> I don't think it's a good idea because then it will be harder for scenario builders to make big section of desert, swamps, forests, etc. on their massive scenario of 200+ tiles in each directions.

Select a 64×64 area, enable land paint mode and you’re done within a jiffy.

**Comment by J-PIE-314** [comment]
I was not able to replicate this behavior, has it been fixed?

**Comment by 733737** [comment, marked as answer]
This has been fixed
